Notes about 10.15 changes (#635)
BohemianCoding/SketchAPI: docs/pages/plugins-notarize-plugins.md

      ```
   2. **Password stored in local Keychain:** You can provide a reference to a **local** keychain item in `.skpmrc` (`skpm` cannot access iCloud keychain items for security reasons). This assumes the keychain holds a keychain item named `AC_PASSWORD` with an account value matching the username `AC_USERNAME`:
      ```yaml
      notarization:
        authority: 'Developer ID Application: Your Team Name'
        username: 'username@org'
        password: '@keychain:AC_PASSWORD'
      ```
      You can create the `AC_PASSWORD` keychain item using any of these two methods:
         1. Using the command-line:
            ```bash
            security add-generic-password -a "AC_USERNAME" -w <super-secret-password> -s "AC_PASSWORD"
            ```
         2. Using the _Keychain Access_ application:
            1. Open Keychain Access.
            2. Add new password item named `AC_PASSWORD`.
            3. Set the password using your app specific password.  

> **Quick tip:** To use your own notarization mechanism and integrate with `skpm`,
provide the command in the `.skpmrc` notarization settings:
>
> ```yaml
> notarization:
>   command: './custom-script.sh'
> ```

> > `skpm` will bundle and zip your plugin, run your command automatically, and set the path to the *zipped* plugin bundle as a parameter, e.g. `./notarize-plugin.sh path/to/plugin.zip`.

Reviewer: ```suggestion
> `skpm` will bundle your plugin, create a ZIP archive of it, and run your command automatically, passing the path to the archive as a parameter, e.g. `./notarize-plugin.sh path/to/select-shapes.sketchplugin-1.0.zip`.
```
